src/webui/src/App.tsx

```diff
@@ -17,7 +17,8 @@ interface AppState {
 }
 
 class App extends React.Component<{}, AppState> {
-    private timerId!: number | null;
+    private timerId!: number | undefined;
+    private flag: boolean = true;
```

Comment:
Please use a meaningful name. I can't really understand what this flag does.

Reply:
OK